### ISSUE TITLE
NavBar should always display the complete First Name

### ISSUE THREAD

**Issue by soustab10** [CircuitVerse/CircuitVerse#2893]
**Describe the bug**
The navbar should display the complete first name and should not replace the last few characters by ... . I think it adds to the experience of using the website when his full first Name is shown (e.g. **Soustab**) and not **Sousta...** . 

**To Reproduce**
1. Go to 'https://circuitverse.org/'
2. Login.
3. See error on the navbar(Error is only be seen if your first name exceeds 6 characters)

**Expected behavior**
The navigation bar should show the complete first name of the user irrespective of its length.

**Screenshots**
When first name exceeds 6 characters and when the user has a surname:
![Screenshot 2022-02-04 161454](https://user-images.githubusercontent.com/95632583/152516729-b8568d21-8317-479e-8e1c-4e8694a198dd.png)

When the user only enters his first name(as the username) the error is not seen until 8 characters.
![Screenshot 2022-02-04 162552](https://user-images.githubusercontent.com/95632583/152517255-d1247da8-7ee4-438e-9aa1-eec54c189aa0.png)

Also note that when the  length of the first name of the user is less than 8 characters then output is fine:
![Screenshot 2022-02-04 161407](https://user-images.githubusercontent.com/95632583/152517442-f17e9a42-05b4-498e-894b-ab59e12fdf94.png)

**Additional context**

**Are you working on this issue? No**

**Comment by rachit3006** [comment]
But in that case, the navbar would shift left according to each name and there would be no uniformity and if the name is too long it would not look aesthetically nice.

**Comment by soustab10** [comment]
Yes, it would. But a user on his page, is only aware of his/her name. So the uniformity across different users matters less than his/her name.(in my opinion).
Or better still if his name altogether is replaced with the profile picture which transforms to a menu upon clicking.
Here's a rough mockup what could be done:
![Screenshot 2022-02-04 170325](https://user-images.githubusercontent.com/95632583/152522278-4533833d-d559-49b4-83bc-9d7d56b7f0bc.png)
Suggestions are welcome!

**Comment by arpandesai0** [comment]
The name should be replaced by a keyword like **profie**, so it's same for all.

**Comment by rachit3006** [comment]
I think the profile picture feature would be better. Can I work on this issue if the maintainers are willing to keep it?

**Comment by rachit3006** [comment]
@soustab10 what should be done? Can I work on this issue?

**Comment by vedant-jain03** [comment]
@gr455 @nitin10s

**Comment by soustab10** [comment]
I think the maintainers should decide what to be done? Fix the exisiting issue or create a new dashboard.

**Comment by prakhar3062** [comment]
@vedant-jain03 i may work on this issue

**Comment by vedant-jain03** [comment]
@prakhar3062 Actually this issue is not approved by maintainers, so I suggest not to go with it as it may be closed and will ruin your work. 
I will suggest you to work on other issues.

**Comment by prakhar3062** [comment]
okay thanks for informing me

**Comment by soustab10** [comment]
Thanks for informing.. @vedant-jain03

**Comment by gr455** [comment]
@nitin10s Mind having a look at this?
